docs/src/PolyhedralGeometry/intro.md

```diff
@@ -110,6 +110,26 @@ Type                                   | An `AffineHyperplane` corresponds to...
 `SubObjectIterator{<:Hyperplane}`      | an element of the iterator.
 
 
+## `IncidenceMatrix`
+
+Some methods will require input or return output in form of an `IncidenceMatrix`.
+
+```@docs
+IncidenceMatrix
+```
+
+From the example it can be seen that this type supports `julia`'s matrix functionality. There are also functions to retrieve specific rows or columns as a `Vector` or a `Set` over the non-zero indices.
+
+```@docs
+row_as_set
+row_as_vector
+column_as_set
+column_as_vector
```

Comment:
Thanks for pointing this out! I still changed this a bit so only the specific methods for `IncidenceMatrix` are included. This might help avoid future conflicts for such a generic funtion name.